link-verifier/action.yml

```diff
@@ -20,6 +20,7 @@ inputs:
   exclude-urls:
     description: 'Comma separated list of URLS not to check'
     required: false
+    default: https://www.misra.org.uk/misra-c, https://www.misra.org.uk
```

Comment:
Placing these here just so that if not adding them explicitly it shows up the in action log